tailwind.config.js

```diff
@@ -1,7 +1,10 @@
-module.exports = {
-  purge: [
-    './src/**/*.vue',
-    './src/**/*.js'
+import { resolve } from 'path'
+
+export default {
+  darkMode: 'class',
+  content: [
+    `${resolve(__dirname)}/src/**/*.vue`,
+    `${resolve(__dirname)}/src/**/*.js`,
```

Comment:
`content` is a newer tailwind config option to replace  `purge` but the reason I'm resolving the globs here is to make sure tailwind runs correctly when we run the vite server from the `tailwind-config-viewer` cli.